Makefile

```diff
@@ -32,3 +32,6 @@ lint:
 	@echo installing python dependencies
 	@pip install --use-wheel -r requirements-dev.in
 	@touch $@
+
+canvas-auth.json:
+	echo '{}' > canvas-auth.json
```

Comment:
As you've seen the app crashes if the `canvas-auth.json` file doesn't exist. This wasn't a problem for the tests because the `AuthData` class (which contains this crash) wasn't being tested. But now it is. So I've edited the `Makefile` to create an empty `canvas-auth.json` automatically if it doesn't already exist.